control-plane-operator/controllers/hostedcontrolplane/konnectivity/reconcile.go

```diff
@@ -281,6 +283,7 @@ func buildKonnectivityWorkerAgentContainer(image, host string, port int32) func(
 			host,
 			"--proxy-server-port",
 			fmt.Sprint(port),
+			"--agent-identifiers=default-route",
```

Comment:
I believe this should be default-route=true 

To properly enable

Reply:
example command line:
https://github.com/kubernetes-sigs/apiserver-network-proxy/commit/554a60b5bacec1e3c27f218d10ec004a244b1719#diff-06009d1be092147fe694efde678466f0c2f7771ba92e63ca27b912fe8e846606R78

Reply:
(and you don't specify default-route=true for the agents in the control plane if that makes sense)